source/Plugins.cpp

```diff
@@ -145,7 +160,15 @@ bool Plugins::HasChanged()
 	for(const auto &it : plugins)
 		if(it.second.enabled != it.second.currentState)
 			return true;
-	return false;
+	return oldNetworkActivity;
```

Comment:
Is this variable needed? If a plugin is downloaded it will return true above already.

Reply:
not if it just gets updated